<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SirusPatcherWindow</class>
 <widget class="QMainWindow" name="SirusPatcherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sirus Patcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/window.ico</normaloff>:/icons/window.ico</iconset>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <widget class="WindowHeaderWidget" name="HeaderWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>940</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 245, 255);</string>
    </property>
    <widget class="QLabel" name="WindowIcon">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>4</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icons/window.ico);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="WindowName">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>4</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="BackgroundHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>940</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>BackgroundHeader</zorder>
    <zorder>WindowIcon</zorder>
    <zorder>WindowName</zorder>
   </widget>
   <widget class="QWidget" name="HeaderButtonsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>939</x>
      <y>0</y>
      <width>62</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(179, 245, 255);</string>
    </property>
    <widget class="QPushButton" name="CloseButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
   
	image: url(:/icons/close.png);
}
QPushButton:hover
{
   
	background-color: rgb(255, 84, 84);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>41</width>
       <height>41</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="HideButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
   
	image: url(:/icons/hide.png);
}
QPushButton:hover
{
	background-color: rgb(39, 147, 224);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/icons/hide.ico</normaloff>:/icons/hide.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>41</width>
       <height>41</height>
      </size>
     </property>
    </widget>
    <zorder>HideButton</zorder>
    <zorder>CloseButton</zorder>
   </widget>
   <zorder>HeaderButtonsWidget</zorder>
   <zorder>HeaderWidget</zorder>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WindowHeaderWidget</class>
   <extends>QWidget</extends>
   <header>QtGui/windowheaderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
